PyPortal/PyPortal_Electioncal_US/electioncal_graphics.py

```diff
@@ -112,7 +112,8 @@ def update_time(self):
         date_str = date_format_str % (year, month, day)
         self.date_text.text = "Today is: " + date_str
 
-    def paragrapher(self, text, cut): # pylint: disable=no-self-use
+    @staticmethod
+    def paragrapher(text, cut):
```

Comment:
Did you try this project out on a device with this change? I don't know of a specific issue with changing to staticmethod but think it's worth a quick check on device to make sure it doesn't cause trouble.

Reply:
I didn't run a full test, but did verify the method still works. It's called [here](https://github.com/adafruit/Adafruit_Learning_System_Guides/blob/f23741077ef4b8aa9e2bf5e18e32936446cd0e9f/PyPortal/PyPortal_Electioncal_US/electioncal_graphics.py#L90) and never uses `self` and isn't accessed directly from `code.py`